Project3/styles.css

```
@@ -0,0 +1,111 @@
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 2px solid white;
+    height: 18px;
+    width: 18px;
+    display: inline-block;
+}
+
+/* font-family: 'BenchNine', sans-serif; */
+/* all width 800px 1024px 1120px (form) */
+body {
+    background-color: #163346;
+    align: auto;
+    padding: 16px;
+    margin: auto;
+    max-width: 300px;
+    min-width: 300px;
+    font-family: 'BenchNine', sans-serif;
+}
+
+form {
+    background-color: #1e2019;
+    padding: 8px;
+    max-width: 250px;
+    min-width: 250px
+}
+
+h2,
+label,
+legend,
+p {
+    color: #fff;
+
+}
+
+h2 {
+    font-family: 'Teko', sans-serif;
+    font-weight: 500;
+    font-size: 35px;
+    line-height: auto;
+    text-align: left;
+    border-bottom: #163346 solid 3px;
+
+}
+
+label,
+legend,
+p {
+    font-family: 'BenchNine', sans-serif;
+    font-size: 18px;
+    font-weight: 800px;
```

`font-weight` values can be either keywords (like `bold`) or values of 100s (like `300` or `400`). In the case of the latter, they do not use a unit of measurement.
```suggestion
    font-weight: 800;
```